src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -67,14 +67,27 @@ public static CompletionItem AddSymbolInfo(IReadOnlyList<ISymbol> symbols, Compl
         {
             var symbol = symbols[0];
             var isGeneric = symbol.GetArity() > 0;
-
             item = item
                 .AddProperty("SymbolKind", ((int)symbol.Kind).ToString())
                 .AddProperty("SymbolName", symbol.Name);
 
             return isGeneric ? item.AddProperty("IsGeneric", isGeneric.ToString()) : item;
         }
 
+        public static CompletionItem AddShouldProvideParenthesisCompletion(CompletionItem item, bool value)
```

Comment:
We don't need add this to the property bag is the value is false.

Reply:
Alternatively, you can consider using `CompletionItemFlags` 
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/CompletionItemFlags.cs,106d32119b01cb47,references